### DISCUSSION TITLE
Building golang functions fails `go mod tidy`

### DISCUSSION THREAD

**Discussion by ededejr** [vercel/community#3644] [Bug Report]
### Page to Investigate

https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/go

### Steps to Reproduce

Hello! I've been building standalone APIs using golang functions on Vercel. On some deployments (and locally as well), I get the following error when `go mod tidy` is run.

```
routes/node_modules/@vercel/go/dist: import path "routes/node_modules/@vercel/go/dist" should not have @version
failed to `go mod tidy`
Error: Command failed: go mod tidy
```

Locally I can fix this by running `go mod tidy` before installing `node_modules` or by deleting `node_modules` but this is blocking by deployments.

Any help would be appreciated 🙂

**Comment by akiomik** [comment]
Try remove `vercel` dependency from `package.json`. If you'd like to use `vercel` command in npm script, you can use `npx vercel` instead.

**Reply by ededejr** [reply]
This did it! Thank you

**Reply by theCompanyDream** [reply]
Hit this problem this fixed this build thank you
